File: Top K Frequent Elements.js
var topKFrequent = function(nums, k) {
    


let res = {};
let a = [];
let p =[];
nums.forEach(element=>{
  if(res[element]){
    res[element]++
  }
  else {
    res[element]=1;
  }
})
let sorted = Object.entries(res).sort((a,b)=>b[1]-a[1])
for(let i = 0 ; i<sorted.length;i++){
 a.push(sorted[i])
 if(a.length==k)break;
}

   
    a.forEach(function (object) {
  var key = Object.keys(a)[0];
  p.push(Number(object[key]))
});;

return p;

};

File: 347-top-k-frequent-elements/top-k-frequent-elements.py
class Solution(object):
    def topKFrequent(self, nums, k):
        hashmap = {}
        res=[]
        for i in nums:
            hashmap[i] = hashmap.get(i,0) +1
        
        return [item[0] for item in sorted(hashmap.items(), key=lambda x: x[1], reverse=True)[:k]]


        
        